### PR TITLE
CVSB-3784

### PR DESCRIPTION
**Pull request by mdclarke21**

### CODE REVIEW

File: src/pages/testing/test-creation/country-of-registration/country-of-registration.ts [hunk 1]
```diff
@@ -55,13 +55,11 @@ export class RegionReadingPage implements OnInit {
   setVehicleRegCountry(regCountryItem) {
     this.focusOut = false;
     this.vehicle.countryOfRegistration = regCountryItem.key;
-    this.visitService.updateVisit();
     this.events.publish(APP.NAV_OUT);
-    this.searchVal = '';
```
Comment (vladddragomir) [inline src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: why did you remove these two lines?

Comment (mdclarke21) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: I asked Ryan about the behaviour of searching for a country. If you searched for example for France and selected France the search list would reset and although France is selected you would not see this on the screen as it has jumped below the fold.

Comment (vladddragomir) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: Is this also specified in some AC? Because I remember the behaviour should have been the actual one.

Comment (mdclarke21) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: Let me check with one of the BAs :)

Comment (vladddragomir) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: great! thanks :)

Comment (mdclarke21) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: @vladddragomir I had a chat with Rav this morning.

So there is no AC that relates directly to the behaviour of what happens when you select a country from the filtered search list: https://jira.i-env.net/browse/CVSB-995.

She is happy with these new changes to go through with this bug, however we can raise an improvement ticket if you feel this is needed :)

Comment (vladddragomir) [reply src/pages/testing/test-creation/country-of-registration/country-of-registration.ts]: Okay @mdclarke21. If you guys are ok with this I am also ok. The fact was not to change an existing behaviour without any discussion with a BA. If Rav agreed this, it's more than ok for me. Cheers!
